### PR TITLE
add (send|receive)_bytes functions

### PR DESCRIPTION
**Pull request by Nudelmeister**

### CODE REVIEW

**Comment by rlane** [comment]
Could you also update shared/api/README.md?

**Comment by Nudelmeister** [comment]
Done. I also updated the quick reference.

File: frontend/app/src/documentation.rs [hunk 1]
```diff
@@ -110,6 +110,8 @@ pub fn documentation(props: &DocumentationProps) -> Html {
               <li><code>{ "get_radio_channel() -> usize" }</code>{ ": Get the radio channel." }</li>
               <li><code>{ "send(data: [f64; 4])" }</code>{ ": Send a message on a channel." }</li>
               <li><code>{ "receive() -> Option<[f64; 4]>" }</code>{ ": Receive a message from the channel. The message with the strongest signal is returned." }</li>
+              <li><code>{ "send_bytes(data: &[u8])" }</code>{ ": Send a message on a channel as bytes, the data will be zero-filled or truncated to a length of 32 bytes." }</li>
+              <li><code>{ "receive_bytes() -> Option<[u8; 32]>" }</code>{ ": Just like receive, but insted the message will be returned as a byte array." }</li>
```
Comment (rlane) [inline frontend/app/src/documentation.rs:114]: Typo on "insted" here and in the next chunk.

**Review by rlane — APPROVED** [review]

File: frontend/app/src/documentation.rs (hunk 1 above)
Comment (Nudelmeister) [reply frontend/app/src/documentation.rs]: fixed it

**Comment by rlane** [comment]
Thanks!

**Pull request merged**